server/pulp/server/db/manage.py

```diff
@@ -191,6 +196,34 @@ def main():
         options = parse_args()
         _start_logging()
         connection.initialize(max_timeout=1)
+
+        active_workers = status.get_workers()
+        if active_workers:
+            last_worker_time = max([worker['last_heartbeat'] for worker in active_workers])
+            time_from_last_worker = UTCDateTimeField().to_python(datetime.now()) - last_worker_time
+            wait_time = timedelta(seconds=constants.MIGRATION_WAIT_TIME) - time_from_last_worker
+
+            if wait_time > timedelta(0):
+                print _('The following processes might still be running:')
+                for worker in active_workers:
+                    print _('\t%s' % worker['name'])
+
+                for i in range(wait_time.seconds, 0, -1):
+                    print _('\rPlease wait %s seconds while Pulp confirms this.' % i),
+                    sys.stdout.flush()
+                    time.sleep(1)
+
+                still_active_workers = [worker for worker in status.get_workers() if
+                                        worker['last_heartbeat'] > last_worker_time]
+
+                if still_active_workers:
+                    print >> sys.stderr, _('\r\nThe following processes are still running, please'
```

Comment:
Is the `\r` needed here?
Also since it's not using `print as a function I think it's not including the automatic newline. Can a line of clean whitespace appear before this text? I think making `\r\n`  ->  `\n\n` would do it.